packages/@okta/i18n/src/properties/login.properties

```diff
@@ -758,6 +758,7 @@ common.logo.alt = aria logo
 ## Generic
 oie.authenticator.enroll.error.fail = Unable to enroll authenticator. Try again.
 oie.authenticator.verify.error.fail = Unable to verify authenticator. Try again.
+oie.form.field.optional = Optional
```

Comment:
I think we may need to generate the ok_PL translations for these new additions.

Reply:
Please add a comment above this line to explain what "Optional" is in reference to (for example: a non-required input field for accepting an additional email address). This provides adequate context for our translators, since "optional" alone is usually not sufficient.